SecretManagement.KeePass.Extension/Tests/Test-SecretVault.Tests.ps1

```diff
@@ -7,8 +7,9 @@ InModuleScope -ModuleName 'SecretManagement.KeePass.Extension' {
             $ModuleName = 'SecretManagement.KeePass'
             $ModulePath = (Get-Module $ModuleName).Path
             $BaseKeepassDatabaseName = "Testdb"
+            $NewLine = [System.Environment]::NewLine
 
-            $KeePassCompositeError = "Exception calling `"Open`" with `"3`" argument(s): `"The composite key is invalid!`r`n`r`nMake sure the composite key is correct and try again.`""
```

Comment:
@trir262 You can use a wildcard, it doesn't have to be explicit.

E.g.`"*The composite key is invalid!*`

That's "good enough", it doesn't have to be a perfect match. I'm not having you match the exception type either :)